### PR TITLE
chore: upgrade to latest docs kit and enable AI assistant

### PR DESCRIPTION
**Pull request by gabriele-ct**

### CODE REVIEW

**Review by FFawzy — APPROVED** [review]
lgtm 👍

File: website-custom-views/gatsby-config.mjs [hunk 1]
```diff
@@ -1,5 +1,7 @@
 // eslint-disable-next-line import/extensions
 import colorPresets from '@commercetools-docs/gatsby-theme-docs/color-presets/index.mjs';
+const { isProd } = process.env;
```
Comment (emmenko) [inline website-custom-views/gatsby-config.mjs:3]: We don't have this defined here in this repo.

**Review by emmenko — CHANGES_REQUESTED** [review]
Thanks!

File: website-custom-views/gatsby-config.mjs (hunk 1 above)
Comment (gabriele-ct) [reply website-custom-views/gatsby-config.mjs:3]: That's fine, I'll remove it and we can stick to the prod url
]

**Review by emmenko — APPROVED** [review]
Thanks 🤗

**Pull request merged**